### ISSUE TITLE
Cannot understand class_weight

### ISSUE THREAD

**Issue by hagianga21** [tensorflow/tensorflow#33076]
**Describe the current behavior**
Hi, how actually does class_weight in model.fit works? Does it multiple to loss function? I cannot find it in source code. If I set class_weight = {0: 0, 1: 0, 2: 0, 3: 0, 4: 0, 5: 0, 6: 0, 7: 0, 8: 1, 9:0}. The loss is not equal 0.

**Comment by gadagashwini** [comment]
@hagianga21,
Please provide the standalone code to reproduce the issue. and also provide the Tensorflow version. Thanks!

**Comment by gadagashwini** [comment]
@hagianga21, Provide more information about issue like code snippet and Tf version.

**Comment by gadagashwini** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
